### DISCUSSION TITLE
Feedback ℹ️

### DISCUSSION THREAD

**Discussion by MystPi** [MystPi/aviate#20] [General]
Share any ideas, issues, problems, etc. you have here. Don't be shy! ✨

**Comment by mybearworld** [comment]
Aviate 2 looks awesome. Also, first comment! 🥳

**Reply by MystPi** [reply]
Thank you, and congrats!

**Comment by mybearworld** [comment]
It might be good to have a page for the statuses, not just the API endpoint.

**Reply by MystPi** [reply]
Yes, I will probably add that eventually since it was included in the previous version of Aviate.

**Reply by Ferretosan** [reply]
couldn't u just copy the code from the old version?

**Reply by MystPi** [reply]
That's... not why I created a new version of Aviate, lol.

**Reply by Ferretosan** [reply]
bruh

**Comment by mybearworld** [comment]
Suggestion: A dark mode 🤔

**Reply by MystPi** [reply]
This was just a test, but I don't think it looks as good as light mode:
<details>
<summary>
Screenshots
</summary>
<img src="https://github.com/MystPi/aviate/assets/86574651/2eb4f288-1e3b-4927-9254-1db5a7a2a365" />
<img src="https://github.com/MystPi/aviate/assets/86574651/a89cca99-98c8-4c1f-81f2-9c69565b2213" />
</details>

**Reply by MystPi** [reply]
I think I would want it to be *just* light mode or *just* dark mode, not both. Maintaining two themes would be a big pain and would make styling a lot more difficult.

Which theme do you think is better?

**Reply by mybearworld** [reply]
I actually really like how that dark theme looks! I would personally go with that.

**Reply by MystPi** [reply]
I'll think about it! It should improve a lot as well since it's not actually finished yet.

**Reply by Ferretosan** [reply]
i also agree that the dark theme test that you created is good

**Comment by thatfrozenfrog** [comment]
Glad to see how far this project was gone:P
I still remember 2 years ago when it was aviate.eu.org and I asked how did you get that domain lol
Anyways your site looks impeccable! Keep up the great work! 🥳🥳

**Reply by MystPi** [reply]
Ah thank you so much!

**Comment by Thebro57** [comment]
It could be a good idea to add functions - y'know, like custom blocks in scratch - so I can get some recursion going and make a xorshift cipher >:)))

**Reply by MystPi** [reply]
That's a nice idea and maybe something I could add to https://github.com/MystPi/scratchstatus.
